src/api/__tests__/email_send.test.js

```diff
@@ -63,7 +63,7 @@ describe('EmailSendHandler', () => {
       {},
       (err, res) => {
         expect(err).toBeNull()
-        expect(res).toBeUndefined()
```

Comment:
@oed Note I changed these tests to match your comment (return success in every cases). Not sure if the dapp relies on the same behavior.

Reply:
I would be surprised if it is :)